using-timescaledb/backup.md

```diff
@@ -1,8 +1,21 @@
 # Backup & Restore [](backup)
 
-Backing up TimescaleDB takes advantage of the reliable functionality already available
-through PostgreSQL.  We recommend using one of two methods depending on your setup:
-`pg_dump` with `pg_restore` or the Write-Ahead Log (WAL).
+Backing up TimescaleDB takes advantage of the reliable functionality already
+available through PostgreSQL.  There are several ways to accomplish this,
+physical backups with `pg_basebackup` or another tool, or logical backups with
+`pg_dump` and `pg_restore`. Physical backups may also be used with Write-Ahead Log
+(WAL) archiving to achieve an ongoing backup.
+
+## Performing Physical Backups [](physical-backups)
+
+For full instance physical backups (which are especially useful for starting up
+new [replicas][replication-tutorial]), [`pg_basebackup`][postgres-pg_basebackup]
+works with all TimescaleDB installations. You can also use any of several
+external backup and restore managers such as [`pg_backrest`][pg-backrest],
+[`barman`][pg-barman], or [`wal-e`][wale official]. These allow you to take
```

Comment:
"take" => "make"?  Unless you mean to change "backups" later in sentence to "snapshots"?

Reply:
Hmmm...I think take sounds better here.